### PR TITLE
Add tracking text when tracking latest price

### PR DESCRIPTION
**Pull request by rjmcneill**
Relates to #651

### CODE REVIEW

**Comment by djmiley** [comment]
Looks good to me, is the css what is specified by UX?

**Comment by rjmcneill** [comment]
@djmiley they showed that the actual label should be in line with the left of the arrow, but it didn't look right like that.
I made it aligned to the center of it.  Happy to change it if people disagree though!

**Comment by jdhodges** [comment]
Do we need a bigger gap between the button and the text? 
image image
Gap seems to be a bit larger on the UX designs.

**Comment by rjmcneill** [comment]
@jdhodges - updated.
That's a far as we can go I reckon

File: src/assets/styles/navigator.less [hunk 1]
```diff
@@ -77,23 +77,35 @@
...
+
+      .reset-button > path {
+        cursor: pointer;
       }
     }
 
-    .reset-button > path {
-      cursor: pointer;
+    .tracking-text {
+      margin: 1px 0 0 7px;
```
Comment (jdhodges) [inline src/assets/styles/navigator.less]: How about:
```margin: 1px 0 -4px 7px;```
Not sure if having a negative margin value is good practice but it gives us a bigger space that's closer to the UX design.

**Comment by rjmcneill** [comment]
Current PR implementation (zero bottom margin):
zeromargintrackingtext

Suggested implementation (negative bottom margin):
minusmargintrackingtext

I quite like the current implementation because it lines the text up with the axis, but I'll change it if we want to increase the gap.

**Comment by rjmcneill** [comment]
@jleft can this be closed now with the #702 PR the new suggestion from UX?

**Comment by jleft** [comment]
> @jleft can this be closed now with the #702 PR the new suggestion from UX?

I think we may still want this - even if it's something which we disable for now.

**Pull request closed without merging**